pallets/committee/src/lib.rs

```diff
@@ -380,6 +389,42 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        #[pallet::weight(10_000)] // TODO: Set weights
+        /// Set new constituent members
+        ///
+        /// This call can only be triggered by `fn propose_constituents` momentary
+        /// after approving of `propose_constituents`
+        pub fn set_members(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
+            Ok(().into())
+        }
+
+        #[pallet::weight(10_000)] // TODO: Set weights
+        /// Propose new constituents to the constituent committee
+        ///
+        /// * This call only can be called by `MemberType::Council`
+        /// * This call will propose the call `set_members`
+        pub fn propose_constituents(
+            origin: OriginFor<T>,
+            _constituents: Vec<AccountIdFor<T>>,
+        ) -> DispatchResultWithPostInfo {
+            let _selector = Self::ensure_member(origin);
```

Comment:
I believe the decision to only allow council members to nominate new members is correct.
But `ensure_member` also succeeds for `Member::Constituent`, so we need to either `ensure!` that the member type is `Council` or add another ensure_method for `ensure_council`.

Reply:
origin check is in the next commits, `ensure_member` for abstracting functions